ui/src/tool-panel/location-track/dialog/location-track-edit-dialog.tsx

```diff
@@ -231,17 +231,29 @@ export const LocationTrackEditDialog: React.FC<LocationTrackDialogProps> = (
     const getDuplicateTrackOptions = React.useCallback(
         (searchTerm: string) =>
             debouncedSearchTracks(searchTerm, props.publishType, 10).then((locationTracks) =>
-                locationTracks
-                    .filter((lt) => {
-                        return lt.id !== props.locationTrack?.id && lt.duplicateOf === undefined;
-                    })
-                    .map((lt) => ({
-                        name: `${lt.name}, ${locationTrackDescription(lt)}`,
-                        value: {
-                            type: 'locationTrackSearchItem',
-                            locationTrack: lt,
-                        },
-                    })),
+                getLocationTrackDescriptions(
+                    locationTracks.map((lt) => lt.id),
+                    'DRAFT',
```

Comment:
Miten tässä menee nyt publishType kun ylempänä on propsuissa annettuna?

Reply:
Käytännössähän edit-dialogissa pitäisi aina olla DRAFT-modessa, mutta rumahan tämä kyllä on näin silti. Vaihdan tämän käyttämään propsuissa kulkevaa publishtypeä, jonka olemassaolon jotenkin missasin alunperin